src/userStatusOverview/types.ts

```diff
@@ -1,26 +1,36 @@
+export enum UserStatusOverviewStatus {
+  Offline,
+  Working,
+  Break,
+  Leave,
+  CoreTimeViolation = 5,
+}
+
 export type UserStatusOverview = {
-  user_id?: number;
+  user_id: number;
   /**
    * Time tracking id of a running task
    */
-  time_tracking_id?: number;
-  time_tracking_start_time?: string;
-  time_tracking_start_time_timezone_id?: string;
-  time_tracking_task_id?: number;
-  time_tracking_is_nonworking?: boolean;
+  time_tracking_id: number;
+  time_tracking_start_time: string | null;
+  time_tracking_start_time_timezone_id: string | null;
+  time_tracking_task_id: number;
+  time_tracking_is_nonworking: boolean | null;
   /**
    * 1 if there is core time violation, default 0
    */
-  core_time_violation?: boolean;
-  core_time_start?: string;
-  updated?: string;
-  is_running?: boolean;
+  core_time_violation: boolean;
+  core_time_start: string;
+  updated: string;
+  is_running: boolean;
   /**
    * Current absence ids
    */
-  current_absence_ids?: string;
+  current_absence_ids: string | null;
   /**
    * Represents current status of users activity if he is offline (0), working(1),break(2),on leave(3), core time violation (5)
    */
-  status?: number;
+  status: UserStatusOverviewStatus;
```

Comment:
Do we know why these two fields were not part of the generated type?

Another question is why do we need it here anyway? is this information not saved in the user type?

Reply:
We are guessing it's not in Swagger then, but we would need to ask ScrummyBears how they generated it. 
It is, but it is in this endpoint too, maybe to avoid having to do another request on clients that might not have all users stored.

Reply:
Okay...  anyway, we should not use them in the react app, because there we want to get all the usernames always from our userStore.